relay-server/src/services/store.rs

```diff
@@ -663,17 +663,6 @@ impl StoreService {
                 );
                 return Ok(());
             }
-            MetricNamespace::Profiles => {
-                if !self
-                    .global_config
-                    .current()
-                    .options
-                    .profiles_function_generic_metrics_enabled
-                {
-                    return Ok(());
```

Comment:
Don't we actually want to always drop profile metrics now? I.e. should we keep the namespace as `Deprecated1` o.a. and always `return Ok(())` for it?